File: java/8.5/example9.java
import java.util.Scanner;

public class example9 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the first number");
        Integer number1 = Integer.parseInt(scanner.nextLine());
        System.out.println("Enter the first second");
        Integer number2 = Integer.parseInt(scanner.nextLine());
        System.out.println("Enter the first third");
        Integer number3 = Integer.parseInt(scanner.nextLine());
        if (number1 > number2 && number1 > number3) {
            System.out.println(number1);
        } else if (number2 > number3) {
            System.out.println(number2);
        } else {
            System.out.println(number3);
        }
    }
}

File: rust/src/8.5/example9.rs
fn main() {
    let mut number1s = String::new();
    let mut number2s = String::new();
    let mut number3s = String::new();
    println!("Enter a number");
    std::io::stdin().read_line(&mut number1s).unwrap();
    println!("Enter a number");
    std::io::stdin().read_line(&mut number2s).unwrap();
    println!("Enter a number");
    std::io::stdin().read_line(&mut number3s).unwrap();
    let number1 = number1s.trim().parse::<i32>().unwrap();
    let number2 = number2s.trim().parse::<i32>().unwrap();
    let number3 = number3s.trim().parse::<i32>().unwrap();
    if number1 > number2 && number1 > number3 {
        println!("{}", number1);
    } else if number2 > number3 {
        println!("{}", number2);
    } else {
        println!("{}", number3)
    }
}